### ISSUE TITLE
🚀 Feature: Documentation

### ISSUE THREAD

**Issue by levino** [stonith404/pocket-id#88]
### Feature description

You should write and host a simple documentation about the features, endpoints, capabilities, etc. Currently I cannot find any.

### Pitch

It would probably make more people use the product if they know what to expect in regards to functionality.

**Comment by stonith404** [comment]
Yeah that's a good idea. Setting it up will take some time, but I’ll look into it as soon as I have some free time.

**Comment by levino** [comment]
I can help and make a basic set up. Which solution would you like? Docusaurus is nice, I find and I know it well. Essentially, one just writes markdown. One could also just extend the markdown files in the docs folder for the time being. Docusaurus can render it as a standalone page nicely later with further ado.

**Comment by stonith404** [comment]
It would be great if you could set the documentation up :) 

I would also suggest to use Docusaurus because we are using it in [Pingvin Share](https://github.com/stonith404/pingvin-share) too. I would add a documentation folder to the root of the repository containing Docusaurus.

Thank you, I'm looking forward to it.

**Comment by kmendell** [comment]
Initial Documentation has been setup here: [stonith404.github.io/pocket-id](https://stonith404.github.io/pocket-id)

**Comment by levino** [comment]
Very nice! Great work.
